ports/qtbase/vcpkg.json

```
@@ -62,7 +62,6 @@
     "icu",
     "jpeg",
     "network",
-    "opengl",
```

You can also use platform expressions here
```suggestion
{
  "name": "opengl",
  "platform": "!ios"
},
```

---

And remove this part instead?

https://github.com/microsoft/vcpkg/pull/36003/files#diff-d285619e22ce186bd6d68cae7da0442cd759a9fd0e0234b253e7f06a4b4f19a9R148-R155

---

Yes

---

Thanks